packages/flutter_tools/test/general.shard/build_system/filecache_test.dart

```diff
@@ -29,6 +31,8 @@ void main() {
     fileSystem.directory('build').createSync();
     environment = Environment.test(
       fileSystem.currentDirectory,
+      artifacts: MockArtifacts(),
```

Comment:
Like here. Log messages from this `Environment` won't go into the `logger` above.

Reply:
Fixed